README.md

```diff
@@ -1,4 +1,4 @@
-<!---
+# <!---
```

Comment:
wrong format, Not need `#`

Reply:
@liuxunorg thanks for catching this issue, just updated.